docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+app:
+  build: ./
+  volumes:
+    - ./:/code
+  links:
+    - database
+  working_dir: /code
+
+database:
+  image: mysql:5
```

Comment:
Absolutely, just didn't know which one to use. Existing tags can be seen on docker hub: https://hub.docker.com/_/mysql/